### ISSUE TITLE
Space between \micro-prefix and rest of unit symbol with siunitx

### ISSUE THREAD

**Issue by mhumbel** [brucemiller/LaTeXML#2371]
LaTeXML produces a  *U+2026 Invisible Times* character between the `\micro` prefix and the rest of the unit symbol when using siunitx. This happens only for `\micro` and not for the other decimal prefixes.

While Chrome seems to display this correctly (i.e. not at all), in Firefox it gets rendered as a space. Although one could argue that this is actually a bug in Firefox, I would still advocate for changing the behavior on the LaTeXML side. I have two arguments for this:

1. Consistency: The *U+2026 Invisible Times* character gets inserted only after the micro-prefix and none of the other decimal prefixes.
2. Semantics: I would consider a unit with a decimal prefix more a compound symbol than a multiplication of two symbols.

Minimal working example:
```
\documentclass{article}
\usepackage{siunitx}
\begin{document}
\si{\micro\meter}
\end{document}
```
The relevant part in the HTML output is:
```
<math id="p1.m1" class="ltx_Math" alttext="\mathrm{\SIUnitSymbolMicro m}" display="inline"><mrow class="ltx_unit"><mi mathvariant="normal">µ</mi><mo>⁢</mo><mi mathvariant="normal">m</mi></mrow></math>
```
with a *U+2026 Invisible Times* character inside the `<mo></mo>`.

I used
 * LaTeXML version 0.8.6
 * Firefox 126.0.1
 * Chrome 126.0.6478.55

**Comment by dginev** [comment]
This sounds related to #2303 . 

If you use the new `--noinvisibletimes` option for `latexmlc`, you may see that behavior mitigated. Currently latexml's default MathML treatment is to err on the side of invisibly multiplied terms.

That said, requesting a binding enhancement is also reasonable, so we should leave the issue open for a bit.

**Comment by teepeemm** [comment]
I agree with the consistency, but I come down on the other side of semantics: I would consider `\micro` to be a prefix that means `10^{-6}`, and so should be multiplied by the base unit (and that this should happen between any prefix and unit).  I think I would also argue for multiplication of some variety between units (eg, in `\si{\newton\meter}`), and not just a non-breaking space.

**Comment by dginev** [comment]
@teepeemm which downstream application of latexml will make use of this semantic distinction? 

I know screen readers (in particular MathCAT) use the invisible characters when voicing the expressions. And "micro times meter" sounds more painful than "micro meter" to me, especially when voiced often.

**Comment by car222222** [comment]
A mathematician writes: this is not multiplication and therefore should not be voiced as if it were. 
Thus Deyan is 'correct' here.

**Comment by teepeemm** [comment]
I still think the prefix is multiplication.  On the other hand, I don't know of anything that would make use of the distinction, and I agree that voicing an invisible character is undesirable.  I'll accede to the majority here.

**Comment by xworld21** [comment]
I think the crux is that this is a linguistic question, not a mathematical question. To keep the two perspectives aligned, one should probably think of 'micro' as a prefix operator, rather than a number which needs to be multiplied. In fact 'micro' on its own does not mean '10^-6', but rather '10^-6 whatever comes next'.

**Comment by teepeemm** [comment]
I looked at this while working on #2784, and I think it comes down to parsing:
* `$\si{\micro\meter}$` becomes `$\mathrm{\SIUnitSymbolMicro m}$` becomes a product
* `$\si{\centi\meter}$` becomes `$\mathrm{cm}$` does not become a product
* `$\SIUnitSymbolMicro m$` becomes a product
* `$cm$` becomes a product

**Comment by car222222** [comment]
Definitely something wrong here then!
